test/menuFactory.spec.js

```diff
@@ -413,5 +413,30 @@ describe('menuFactory', () => {
       ReactDOM.render(<Menu isOpen />, container);
       expect(component.state.isOpen).to.be.true;
     });
+
+    it('will not change when isOpen prop was not changed', () => {
+      let container = document.createElement('div');
+      let ParentComponent = React.createClass({
+        getInitialState() {
+          return { collapsed: true };
+        },
+        changeProps() {
+          this.setState({
+            collapsed: !this.state.collapsed
+          });
+        },
+        render() {
+          return (
+            <Menu ref="menu" {...this.state} />
+          );
+        }
+      });
```

Comment:
Yep, did that before merging https://github.com/negomi/react-burger-menu/commit/64f30b2be24ecfaf2355fad8932668599d2960de :)